Ensure desirable compile flags are used.  We want RPM_OPT_FLAGS rather than
upstream's idea of suitable optimization flags, per Fedora packaging
guidelines, and we *don't* want -ffast-math, per bug #629964.


diff -Naur libdbi-0.8.2.orig/configure.in libdbi-0.8.2/configure.in
--- libdbi-0.8.2.orig/configure.in	2007-02-14 17:51:02.000000000 -0500
+++ libdbi-0.8.2/configure.in	2007-10-30 16:11:59.000000000 -0400
@@ -55,7 +55,7 @@
         case $host in
         *-*-linux*)
                 DEBUG="-g -Wall -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char -std=gnu99"
-                CFLAGS="-O20 -ffast-math -D_REENTRANT -fsigned-char -std=gnu99"
+                CFLAGS="-D_REENTRANT -fsigned-char -std=gnu99 $RPM_OPT_FLAGS"
                 PROFILE="-pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char -std=gnu99";;
         sparc-sun-*)
                 DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char -mv8 -std=gnu99"
